src/Illuminate/Foundation/ComposerScripts.php

```diff
@@ -41,6 +41,17 @@ protected static function clearCompiled()
     {
         $laravel = new Application(getcwd());
 
+        // Create environment file if it doesn't exists.
+        if (! file_exists($laravel->environmentFilePath())) {
+            exec("php -r \"copy('".$laravel->environmentFilePath().".example',
+             '".$laravel->environmentFilePath().');"');
```

Comment:
@crynobone This operation is literally the same as when you create a new laravel application and new .env file is created after installation of all packages. https://github.com/laravel/laravel/blob/master/composer.json#L32-L34